pkg/controller/cincinnati/new.go

```diff
@@ -460,6 +507,20 @@ func (k *kubeResources) newGraphBuilderContainer(instance *cv1beta1.Cincinnati,
 	return g
 }
 
+func (k *kubeResources) newGraphBuilderVolumeMounts(instance *cv1beta1.Cincinnati) *[]corev1.VolumeMount {
```

Comment:
nit: I don't think there's an advantage here to returning the pointer to the slice. The slice already is essentially a reference.

Reply:
Same above in `newVolumes`